### ISSUE TITLE
Hunter Aspect Request

### ISSUE THREAD

**Issue by someolguy** [xod-wow/LiteMount#251]
Can you make it switch back to previous aspect?
In dungeons, I like to give the group the added run speed from one area to another, but it sure would be nice to return to previous aspect (hawk) or perhaps check a box in options for hunter class and choose which one to return to?

Kinda like how ya did the druid shapeshift return?

**Comment by mbattersby** [comment]
I apologize, I don't understand what you mean.

LiteMount doesn't do anything with hunter aspects. It is a mounting addon and deals with mounts and also druid forms that are like mounts.

Is this something you have configured yourself?

**Comment by someolguy** [comment]
well, since you were able to return it to previous form on druids, was thinking doing the same for hunters if ya click macro again
ON: Cheetah
OFF: Hawk

**Comment by mbattersby** [comment]
I'm not planning to add any support for hunter aspects into LiteMount directly.

You can possibly do this with your macro somehow, I'm not a macro expert. If you can figure out how you can put that into your unavailable macro.

You can also do this with the (advanced options)[https://github.com/xod-wow/LiteMount/wiki/Advanced-Options] by editing the action list text and before the `Macro` line at the end put

`
Spell [aura:Aspect of the Hawk] Aspect of the Cheetah
Spell [aura:Aspect of the Cheetah] Aspect of the Hawk
`

**Comment by someolguy** [comment]
So do this?  (did not work)
=================
LeaveVehicle
Dismount [nofalling]
CopyTargetsMount
ApplyRules
IF [mod:shift]
    IF [submerged]
        Limit -SWIM
    ELSEIF [dragonridable]
        Limit -DRAGONRIDING
    ELSEIF [flyable]
        Limit -FLY
    ELSEIF [floating]
        Limit -SWIM
    END
END
SmartMount
IF [falling]
  # Slow Fall, Levitate, Zen Flight, Glide, Flap
  Spell 130, 1706, 125883, 131347, 164862
  # Hearty Dragon Plume, Rocfeather Skyhorn Kite
  Use 182729, 131811
  # Last resort dismount even if falling
  Dismount
END
Spell [aura:Aspect of the Hawk] Aspect of the Cheetah
Spell [aura:Aspect of the Cheetah] Aspect of the Hawk
Macro

**Comment by mbattersby** [comment]
Yeah I would expect that to work. I'm afraid I don't play Cata so I can't help any further than that.
